website_cookiefirst/views/website_template.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template
+        id="footer_copyright_company_name"
+        inherit_id="website.footer_copyright_company_name"
+    >
+        <xpath expr="//span[hasclass('o_footer_copyright_name')]" position="after">
+            <span>-
+                <a href="/legal/cookies-policy">Cookies Policy</a>
+            </span>
+        </xpath>
+    </template>
+
+    <template id="layout" inherit_id="website.layout">
+        <xpath expr="//script[@id='tracking_code']" position="attributes">
+            <attribute name="type">text/plain</attribute>
+            <attribute name="data-cookiefirst-category">performance</attribute>
+        </xpath>
+        <xpath expr="//script[last()]" position="after">
+            <t t-if="website.cookiefirst_identifier">
+                <script
```

Comment:
After setting cookiefirst_identifier I can't find this script on the website

Reply:
![screenshot](https://github.com/user-attachments/assets/f6686b5f-bbdf-4a30-bf18-b4c6ceebe19c)
it is there. have you setup the domain correct at cookiefirst?

Reply:
Weird I don't have it. I would expect the script to be there even if the value is misconfigured because it only checks if the value is truthy. I don't think that banner.js removes itself if it's erroneous, correct me if I'm wrong (it's not in the page source).
Testing it on runboat btw

Reply:
Okay thanks! I was yesterday late at night trying to figure out why it's not loaded :smile:  Should we add a note in the readme about this do you think?

Reply:
Yeah I think this will help if somebody is facing this issue. Will add a note in the readme.